platform-hub-api/app/models/identity.rb

```diff
@@ -10,6 +10,8 @@ class Identity < ApplicationRecord
     kubernetes: 'kubernetes'
   }
 
+  has_many :tokens, -> { where kind: 'user' }, as: :tokenable, class_name: KubernetesToken
```

Comment:
Something doesn't seem right about having this association in `Identity` directly, given it only applies to one type of identity object – what do you think about removing it from here, and just using the querying/scopes on `KubernetesToken` directly?

Reply:
I know it's only relevant to kubernetes identity, however having this association simplifies creation of tokens for given user as it automatically sets tokenable object. Can remove and modify other places but I don't think it'll make things dramatically better. Perhaps just a comment here should suffice?

Reply:
Yeah that's fine. I would say that we should add a validation to ensure only `kubernetes` identities actually ever have tokens, but it's fine as-is for now.